ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1328,5 +1328,26 @@
     "CaseEventID": "attachScannedDocs",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "notifyGatekeeper",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "notifyGatekeeper",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "notifyGatekeeper",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "R"
```

Comment:
are we not changing live from ?

Reply:
Don't believe so, that is on of the fields that is just a constant throughout all of the definitions. Although thinking about it may be able to be used for feature toggling purposes 🤔

Reply:
okay 👍 i will make a note of it